srcpkgs/qtlanguageserver/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'qtlanguageserver'
+pkgname=qtlanguageserver
+version=6.5.0
+revision=1
+build_style=cmake
+hostmakedepends="qt6-base"
+makedepends="qt6-base-devel"
+depends="qt6-base"
+short_desc="Implementation of the Language Server Protocol for Qt6"
+maintainer="Adam Pschorr <adampschorr13@aol.com>"
+license="custom:LicenseRef-Qt-Commercial, LGPL-3.0-only, GPL-2.0-only, GPL-3.0-only"
+homepage="https://github.com/qt/qtlanguageserver"
+distfiles="https://github.com/qt/qtlanguageserver/archive/refs/tags/v$version.tar.gz"
+checksum=10b57ceb8a893089d396383d340bafdb75b364f332d8390f7b90f34092f706c5
+
+post_install() {
+	for License in $wrksrc/LICENSES/*; do vlicense $License; done
```

Comment:
only `LicenseRef-Qt-Commercial` needs to be installed out of the ones you listed above, and you didn't list everything in that dir in the `license` variable